spec/fixtures/1849/closure.json

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
delete this spec and use a real game spec in the future

Reply:
sure